examples/shell/cmd_device.cpp

```diff
@@ -21,19 +21,13 @@
 
 #include <platform/CHIPDeviceLayer.h>
 #include <shell/shell.h>
-#include <support/Base64.h>
 #include <support/CHIPArgParser.hpp>
+#include <support/CHIPMem.h>
```

Comment:
truth be told I'm not sure this is better, because now we have to verify it's not leaking...